bots/github/cache.py

```
@@ -48,10 +48,13 @@ def prune(self):
         if not os.path.exists(self.directory):
             return
         now = time.time()
-        for filename in os.listdir(self.directory):
-            path = os.path.join(self.directory, filename)
-            if os.path.isfile(path) and os.stat(path).st_mtime < now - 7 * 86400:
-                os.remove(path)
+        try:
+            for filename in os.listdir(self.directory):
+                path = os.path.join(self.directory, filename)
+                if os.path.isfile(path) and os.stat(path).st_mtime < now - 7 * 86400:
+                    os.remove(path)
+        except OSError:
+            pass
```

It would be better to only wrap the `os.remove` into the `try:`, as otherwise this might hide errors in the loop and stating.